bitcoin/src/blockdata/script/push_bytes.rs

```diff
@@ -139,15 +162,15 @@ mod primitive {
                     fn from(bytes: &'a [u8; $len]) -> Self {
                         // Check that the macro wasn't called with a wrong number.
                         const _: () = [(); 1][($len >= 0x100000000u64) as usize];
-                        // We know the size of array statically and we checked macro input.
+                        // SAFETY: We know the size of array statically and we checked macro input.
                         unsafe { PushBytes::from_slice_unchecked(bytes) }
                     }
                 }
 
                 impl<'a> From<&'a mut [u8; $len]> for &'a mut PushBytes {
                     fn from(bytes: &'a mut [u8; $len]) -> Self {
                         // Macro check already above, no need to duplicate.
-                        // We know the size of array statically and we checked macro input.
+                        // SAFETY: We know the size of array statically and we checked macro input.
```

Comment:
Perhaps make a separate commit adding these "SAFETY` prefixes (but does it need to be all caps? I believe "Safety" works too) and possibly other polishes.

Reply:
Writing SAFETY in all caps is a conventions, you see it all around the standard library, also here in this [clippy lint](https://rust-lang.github.io/rust-clippy/master/index.html#/undocumented_unsafe_blocks) for example.

Reply:
I knew about the lint but I thought it's case-insensitive. But OK, if people use all-caps let's do it too.